convertAngle.ts

```diff
@@ -0,0 +1,52 @@
+import { assert } from "testing/asserts.ts";
```

Comment:
Import `assert` from `"./except.ts"`. The `assert` from `"testing/asserts.ts"` is only used in the tests and won't be available in the Node.js / Browser versions.
Also, you need to format the code again. Please always format the code before committing anything.

Reply:
Ok I changed the `assert` import, and with that I was able to use `TypeError` again, instead of `AssertionError`.

But I did the formatting before last commit. Even now doing `deno lint ./*.ts` doesn't change anything. It just says `Checked 93 files`. Am I missing something ?

PS I also run tests, all succeeded.

Reply:
Did it. I had to use `deno fmt` instead of `deno format` because this one gave me an "argument not expected" error.